core/services/chainlink/testdata/config-multi-chain-effective.toml

```diff
@@ -207,6 +207,14 @@ OCRDevelopmentMode = false
 InfiniteDepthQueries = false
 DisableRateLimiting = false
 
+[Tracing]
+Enabled = false
+CollectorTarget = ''
+NodeID = ''
+SamplingRatio = 1.0
+
+[Tracing.Attributes]
```

Comment:
Is `omitempty` not working as expected?

Reply:
I don't this approach is handling it it's still getting populated like this

Reply:
I wonder if a named type instead of a raw map would help?

Reply:
Locally it helped, but it caused some other weirdness with a `user` field just hanging around at the bottom of the generated TOML. I timeboxed it at 20 minutes, but I can continue to play around with it later or on a different PR if you don't think it's blocking.

Reply:
I think we need to understand what is happening because this has not been a problem so far IIRC. Aren't we using maps in other places too?

Reply:
I don't actually see any other map types. All our other TOML tables use structs.

> it caused some other weirdness with a user field just hanging around at the bottom of the generated TOML.

What do you mean? Like invalid TOML?

Reply:
Some fiddling in the go playground suggests that a nil map or a nil pointer to a map will already be omitted in the output by default, without the `omitempty` struct tag. Adding the tag additionally omits output for zero values in the non-pointer case, but not for the pointer case! I think this is by design, because a zero value for pointer to a map is nil, not an empty map. So I think you can just do non-pointer: `Attributes map[string]string`.